include/nil/blueprint/non_native_marshalling.hpp

```diff
@@ -197,6 +197,44 @@ namespace nil {
             return value_into_vector<BlueprintFieldType, NonNativeFieldType>(typename NonNativeFieldType::value_type(glued_non_native));
         }
 
+        /*template<typename BlueprintFieldType>
+        size_t size_of_non_native_type (llvm::GaloisFieldKind arg_field_type) {
```

Comment:
If we do not need this function - maybe we should delete it?

Reply:
thanks, just miss it